drivers/bridge/bridge.go

```diff
@@ -969,7 +971,9 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 	}
 	defer func() {
 		if err != nil {
-			d.nlh.LinkDel(host)
+			if dellinkerr := d.nlh.LinkDel(host); dellinkerr != nil {
```

Comment:
Can we reuse err instead of dellinkerr like above change ?

Reply:
I worry that this will overwrite the previous errorand  the error of LinkDel can be recorded by log